### PR TITLE
Semantic Segmentation

### PR DESCRIPTION
**Pull request by seanytak**
### Generic Semantic Segmentation Training Tools using PyTorch and Azure Machine Learning
Adds the following to the `contrib` section
- Training script for semantic segmentation model using Azure Machine Learning
- Utilities to convert from various semantic segmentation label formats
- Utilities to work with semantic segmentation masks
- PyTorch dataset for semantic segmentation

### Checklist:

- [WIP] I have followed the [contribution guidelines](../CONTRIBUTING.md) and code style for this project.
- [ X This branch is created from `staging` and not `master`.
- [X] This PR is being made to `staging` and not `master`.
- [X] I will squash merge this PR into `staging`.
- [X] I have added tests covering my contributions.
- [WIP] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by PatrickBue** [comment]
Thanks for this contribution. Could you please add some more information what this PR does, how to install, with 1-2 screenshots, etc. For example the segmentation/readme file currently is empty.

**Comment by PatrickBue** [comment]
@seanytak Feel free to ping me should you have any questions or comments.
